modules/ml-training/k8s-managed/stack.py

```diff
@@ -0,0 +1,221 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from typing import Any, cast
+
+import cdk_nag
+from aws_cdk import Aspects, Stack, Tags, aws_eks, aws_iam
+from cdk_nag import NagPackSuppression, NagSuppressions
+from constructs import Construct, IConstruct
+
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+class TrainingDags(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        *,
+        deployment_name: str,
+        module_name: str,
+        mwaa_exec_role: str,
+        eks_cluster_name: str,
+        eks_admin_role_arn: str,
+        eks_openid_connect_provider_arn: str,
+        **kwargs: Any,
+    ) -> None:
+        # ADDF Env vars
+        self.deployment_name = deployment_name
+        self.module_name = module_name
+        self.mwaa_exec_role = mwaa_exec_role
+
+        super().__init__(
+            scope,
+            id,
+            description="(SO9154) Autonomous Driving Data Framework (ADDF) - k8s-managed",
```

Comment:
This is not a part of the guidance..why this description?

Reply:
It's legacy from the k8s managed simulation module. I updated to: ""Autonomous Driving Data Framework (ADDF) - k8s-managed ML training""